openmetadata-ui/src/main/resources/ui/src/constants/Glossary.constant.ts

```diff
@@ -32,3 +33,18 @@ export const ADD_GLOSSARY_ERROR = {
     }) + '.'
   ),
 };
+
+export const STEPS_FOR_IMPORT_GLOSSARY_TERMS = [
+  {
+    name: startCase(t('label.upload-file')),
+    step: 1,
+  },
+  {
+    name: startCase(t('label.preview-uploaded-data')),
+    step: 2,
+  },
+  {
+    name: startCase(t('label.import')),
+    step: 3,
+  },
+];
```

Comment:
Lets update key directly instead of using startCase

Reply:
Thanks, @ShaileshParmar11, I will take care of this in my next patch.